custom_components/easee/controller.py

```diff
@@ -538,6 +592,8 @@ async def async_initialize(self):
                     _LOGGER.debug("Found site (unmonitored): %s %s", site.id, site.name)
                 else:
                     _LOGGER.debug("Found site (monitored): %s %s", site.id, site.name)
+                    cost_data = CostData(site, 60)
```

Comment:
To make it easier to follow
```suggestion
                    cost_data = CostData(site, period=60)
```